pkg/netpol/connlist/connlist.go

```diff
@@ -544,7 +578,7 @@ func (ca *ConnlistAnalyzer) getPeersForConnsComputation(pe *eval.PolicyEngine) (
 	// update the ca.peersList from workload peers list (used for updating dot outputs with all workloads from manifests)
 	ca.peersList = make([]Peer, 0, len(peerList))
 	for _, p := range peerList {
-		if ca.isPeerFocusWorkload(p) {
+		if isPeerFocusWorkload(p, ca.focusWorkloads) || (len(ca.focusWorkloadPeers) != 0 && ca.isPeerFocusWorkloadPeer(p)) {
```

Comment:
if only input arg was focusWorkloadPeers (so that it needs to be ignored), will it be re-assigned as empty list? or will such peers be added to `ca.peersList` even if this flag should be ignored?

Reply:
if the flag should be ignored - means the `focusworkload` is not used which means all the peers should be in the peerlist so no such problem

Reply:
I meant in case that focusworkload  should be used, but only focusworkload -peer should be ignored

Reply:
the goal from this check if there are two lists (focusworkload and focusworkload-peer) to add peers from both to the list.
if focusworkload is empty then all peers need to be in the connlist - and second part of this condition has no meaning